test/Microsoft.ML.Tests/OnnxConversionTest.cs

```diff
@@ -1764,32 +1855,33 @@ private void CompareResults(string leftColumnName, string rightColumnName, IData
         {
             var leftColumn = left.Schema[leftColumnName];
             var rightColumn = right.Schema[rightColumnName];
-            var leftType = leftColumn.Type.GetItemType();
-            var rightType = rightColumn.Type.GetItemType();
+            var leftType = leftColumn.Type.GetItemType().RawType;
+            var rightType = rightColumn.Type.GetItemType().RawType;
+            Assert.Equal(leftType, rightType);
```

Comment:
Why is this changing? I am assuming that it is because ONNX models don't have the concept of key types, but it would make all the other tests less strict as well. Perhaps add a boolean argument to this method, such as `compareItemType`?

Is this the first transformer that outputs key types that can be exported to ONNX? If not, then how is this handled in other transformers? 
If yes, then we should decide how to handle exporting key types, because of the `+1` difference in the outputs.

Reply:
Yes, I added this because ONNX doesn't have a key concept,so I thought it would make more sense to compare raw types. I actually reverted this change, as now I think this could be a standalone PR.